### ISSUE TITLE
lr scheduling with muon

### ISSUE THREAD

**Issue by rishabh-ranjan** [MoonshotAI/Moonlight#34]
Thanks for your work on scaling up Muon training!

For AdamW best results are usually attained with a warmup+decay LR schedule.
Have you guys tried LR scheduling with Muon?
What are the recommended best practices for LR scheduling with Muon?

For example, in torch OneCycleLR (https://docs.pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html),
the `cycle_momentum` is `True` by default.
Should this also be the case for Muon, or is it better to keep momentum fixed even with LR scheduling?

**Comment by MarktHart** [comment]
I've been using Muon as my goto optimizer since November, but have been training small-ish models. Think 100M - 1B parameters. 

In my experience, you need less/no LR warmup with Muon. I mostly go with linear/trapezoid (aka stable then linear) decay.
